setup.py

```diff
@@ -423,7 +423,7 @@ def getExtensionModules(nupicCoreReleaseDir, platform, bitness, cmdOptions=None)
       ["extensions/cpp_region/PyRegion.cpp",
```

Comment:
Need to fixup this and the next line, too, to match `cpp_region` -> `py_algorithms` renaming.